src/InvokePlaster.ps1

```diff
@@ -1,4 +1,4 @@
-<#
+﻿<#
```

Comment:
What editor are you using? If it is VSCode I suggest you add the user setting: `"files.trimTrailingWhitespace": true`. It is a very nice setting for PowerShell work and would eliminate this "apparent" trailing whitespace change.